torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -253,6 +255,9 @@ InputsIdLookup::IdLookupReturn InputsIdLookup::lookupId(
     }
     encoding_.push_back(';');
   }
+  if (additional_info) {
+    encodeBuffer(additional_info->getCommonAlignmentSize(), encoding_);
```

Comment:
@jjsjann123 should this be per input, or do you  think this is okay?

Reply:
Great question and I wanted to discuss this as well. 

Here I was just trying to avoid combinatorically growing number of compiled kernels for multiple input cases if we assign a different id for each alignment case.

It will also depend on our strategy to vectorize the input loads long term. If we go with current `norm_hack` approach where all inputs agree on the same vectorization factor then this caching would be fine but if we decide to have a different alignment-dependent vectorization factor for each input then we will need to cache based on alignments of each input individually and we just need to watch out that it may grow the number of kernel compilations. 

I will change this to adding the alignment size of each input just to be safe for now.